tools/neurochem-test.py

```diff
@@ -1,84 +0,0 @@
-import os
```

Comment:
I already forget what this file is used for, but I don't think it is useful anymore.